applications/crossbar/src/modules/cb_accounts.erl

```diff
@@ -1381,10 +1371,9 @@ set_trial_expires(JObj) ->
     Expires = kz_time:now_s() + TrialTime,
     kzd_accounts:set_trial_expiration(JObj, Expires).
 
-
 -spec load_initial_views(cb_context:context()) -> 'ok'.
 load_initial_views(Context)->
-    _ = kz_datamgr:refresh_views(cb_context:account_db(Context)),
+    _ = kapps_maintenance:refresh(cb_context:account_db(Context)),
```

Comment:
the intent here is to refresh_views, so a call to kz_datamgr:refresh_views seems appropriate since `kapps_maintenance:refresh` may do other things other than refreshing_views